policies/definitions.md

```diff
@@ -0,0 +1,150 @@
+# Glossary of OpenSSL terms and definitions
```

Comment:
Isn't "definitions" kind of implicit in Glossary? i.e. is "Glossary of OpenSSL terms" enough? Or alternatively "Definition of OpenSSL terms"

Reply:
I prefer glossary, this doesn't really define anything it, instead, provides a brief description and a reference to where the term is defined.